src/commands/graph.ts

```diff
@@ -17,7 +17,13 @@ export function graphCommand() {
   const cwd = getCwd()
   if (!cwd) return
 
+  // Get the configured shell command
+  const config = vscode.workspace.getConfiguration(
```

Comment:
Please make this fit on a single line (with the format configuration provided in the repo it should be made automatically using the "format" script

Reply:
👍